backend/app/interfaces/ceqr_data/ceqr_school_buildings.rb

```diff
@@ -13,9 +13,9 @@ def high_schools_in_boro(borocode)
 		# all primary schools (PS, PSIS) within project district and subdistrict
 		def ps_schools_in_subdistricts(subdistrict_pairs)
 			@dataset.select(
-        :geom, :district, :subdistrict, :borocode, :bldg_name, :excluded, :bldg_id, :org_id, :org_level, :name, 
-		    :address, :ps_capacity, :ps_enroll, :ms_capacity, :ms_enroll, :hs_capacity, :hs_enroll, :ogc_fid
-		  ).where(Sequel.lit("(district, subdistrict) IN (VALUES #{subdistrict_pairs.join(',')})")).where(Sequel.ilike(:org_level, '%PS%'))
+        		:geom, :district, :subdistrict, :borocode, :bldg_name, :excluded, :bldg_id, :org_id, :org_level, :name, 
+		    	:address, :ps_capacity, :ps_enroll, :ms_capacity, :ms_enroll, :hs_capacity, :hs_enroll, :ogc_fid
+		  	).where(Sequel.lit("(district, subdistrict) IN (VALUES #{subdistrict_pairs.join(',')})")).where(Sequel.ilike(:org_level, '%PS%'))
```

Comment:
There are a couple column name changes incoming from `ceqr_school_buildings`:
- `ps_capacity` => `pc`
- `ps_enroll` => `pe`
- `ms_capacity` => `ic`
- `ms_enroll` => `ie`
- `hs_capacity` => `hc`
- `hs_enroll` => `he`

Additionally, we need to add the new `source` field, which is where we'll get `bluebook` or `lcgms`

Reply:
Will add!